internal/driver/glfw/window.go

```diff
@@ -501,6 +505,11 @@ func (w *window) SetContent(content fyne.CanvasObject) {
 	}
 
 	w.canvas.SetContent(content)
+	if !visible && w.fixedSize && (w.requestedWidth == 0 || w.requestedHeight == 0) {
+		bigEnough := w.canvas.canvasSize(content.MinSize())
+		w.Resize(bigEnough)
+	}
```

Comment:
This look like a bit of duplicated code to me. Would it be possible to put most of the if tests along with the code inside the if in its own function?

Reply:
Yeah could do. Initially it seemed like longer code - but the duplication is undesirable, will fix